Sources/Rendering/OpenGL/glsl/vtkVolumeFS.glsl

```diff
@@ -465,7 +466,7 @@ vec4 getColorForValue(vec4 tValue, vec3 posIS, vec3 tstep)
 
     // If I am on the border, I am displayed at full opacity
```

Comment:
technically this comment is not 100% correct

Reply:
Sorry, I meant that this comment should be updated because it used to be true "full opacity", but with your change, it may not be "full opacity", it depends on `outlineOpacity`

Reply:
@martinken Thanks for your explanation. So based on my understanding there is no way to have the outlineThickness (in the above images) act linearly to the opacity. Correct me please if I'm wrong.